lxd/instances_get.go

```diff
@@ -304,6 +305,8 @@ func instancesGet(d *Daemon, r *http.Request) response.Response {
 	}
 
 	resultErrListAppend := func(inst db.Instance, err error) {
+		logger.Error("Failed getting instance info", logger.Ctx{"err": err, "instance": inst.Name})
```

Comment:
we should log `project` in the error context too otherwise its not clear which instance we are logging about.

`project:` comes first before `instance:`.

Reply:
Done!